### DISCUSSION TITLE
How can I tip you for the work that you have done? Like is there a PayPal or a patreon where i can tip you?

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#89] [General]
Nagadomi, How can I tip you for the work that you have done? Like is there a PayPal or a patreon where i can tip you?

**Comment by nagadomi** [comment]
I have patreon account. 
https://www.patreon.com/nagadomi
Also have Fantia (mainly for japanese).
https://fantia.jp/waifu2x

**Reply by kaelsonofkrypto** [reply]
thanks, just sent u a lil something on patreon, , my name will show up as Anthony R. appreciate it
